lib/takeaway.rb

```diff
@@ -0,0 +1,41 @@
+require_relative './order'
+require_relative './text'
+
+class Takeaway
+  def initialize(menu = Menu.new, order_class = Order)
+    @menu = menu
+    @order_class = order_class
+    @current_order = nil
+  end
+
+  def show_menu
+    @menu.list_items
+  end
+
+  def start_new_order
+    @current_order = @order_class.new
+  end
+
+  def check_bill
+    no_order?
+    @current_order.bill
+  end
+
+  def add_to_order(dish)
+    no_order?
+    @current_order.add(dish)
+  end
```

Comment:
maybe can find a way to add quantity in there, so that it is easier to order multiple of a single item